features/FxA-54-push-on-status-change/README.md

```diff
@@ -63,9 +63,14 @@ in response to the following actions:
 * User deletes their account
 * User resets their password
 
+When the fxa-auth-server receives a `410 Gone`
```

Comment:
from meeting: we shall clear the `callbackUrl` (sp?) if status is 400 level then we clear the record for push and it is up to device to register. 

for the desktop / client patch: we need to add a check for device callback is not empty.